src/collections/blog/2023/2023-07-22-introduction-to-layer5-resources/index.mdx

```diff
@@ -0,0 +1,98 @@
+
+<div>
+  <h2>What Will You Learn In This Course?</h2>
+
+  <p>The learning path contains a range of topics from the basics to advanced topics on services meshes and the role Meshery plays as a cloud-native management plane. 
+  By the end of this learning path, you will have the knowledge and confidence on how to: 
+  </p>
+
+  <ol>
+    <li>Deploy an application using service meshes(Istio & Linkerd). </li>
+    <li>Expose services using ingress gateway for different service meshes.</li>
+    <li>Understand how service mesh performance is used to measure for Observability in your system</li>
+    <li>With Routing and canary, you will get your hands on some of the traffic management capabilities for different service meshes.</li>
+    <li>You will discover how to use systematic defects to assess an application's resilience.</li>
+    <li>You will gain insight on topics like, fault injection, circuit breaking, mutual Tls and more.</li>
+    <li>You will learn more about Service security capabilities and how that work. </li>
+    <li>Experience huge performance improvements using Webassembly and intelligent data planes.</li>
+  </ol>
+
+  <h2>This course is for you if…</h2>
+  <ul>
+    <li>You want to gain a clear and advanced understanding of Service Meshes.</li>
+    <li>You have been trying to learn and understand service mesh but still don't understand.</li>
+    <li>You are an application architect or developer.</li>
+    <li>You already understand service meshes and are looking for a course to cover advanced topics. This learning path also includes expert topics!</li>
+  </ul>
+</div>
+
+<div>
+  <h2>What's Next?</h2>
+  <p>The upcoming phases of this learning Path/certification will focus on capturing users’ data to help track progress on each learning path. Multiple learning paths, interactive labs, and quizzes will test participants' knowledge and award them a certificate upon completion of a learning path. Maintainers and contributors are working towards implementing some of these features in future releases. Some of the core features to expect:</p>
+
+  <ul>
+    <li>Multiple Learning Paths.</li>
+    <li>User Account</li>
+    <li>Bookmarking / Progression tracking</li>
+    <li>Quizzes and grades</li>
+    <li>Certificate of Completion </li>
+  </ul>
+</div>
+
+<h2>Conclusion</h2>
+<p>Layer5 is a creator and maintainer of service mesh standards and this makes it the best place for you to gain this knowledge.  One of our fantastic initiatives is the interactive practice labs, which let you study while you explore services meshes on your browser. As the primary developers of the cloud native management plane, Meshery, our maintainers are readily available to answer your questions, should yin case you have any. 
```

Comment:
```suggestion
<p>Layer5 is a creator and maintainer of service mesh standards and this makes it the best place for you to gain this knowledge. One of our fantastic initiatives is the interactive practice labs, which let you study while you explore services meshes on your browser. As the primary developers of the cloud native management plane, Meshery, our maintainers are readily available to answer your questions, should in case you have any.
```